### DISCUSSION TITLE
How to edit a recorded route after it is saved?

### DISCUSSION THREAD

**Discussion by sleeplessKomodo** [OpenTracksApp/OpenTracks#1921] [Q&A]
Hey, loving the app via android btw!

I was just curious, if it is at all possible to edit a recorded route after it's saved, because many times, I have mistakenly left the recording running long after I have finished my trip, which leaves a chunk of garbage data at the end of the recorded trip. 

If there was some way to delete the junk data of a trip, say via a range of time with a delete function, that would be super cool. 

If it's as simple as editing some files, then I don't mind doing so, I just need some guidance because I don't want to ruin my files by accidentally editing something I shouldn't touch. 

Thanks in advance!

**Comment by dennisguse** [comment]
Sadly, no. 

See #375. 

The biggest problem is actually building a proper UI for this. 
From a data perspective, everything is in place.
